### ISSUE TITLE
support copying to clipboard as HTML

### ISSUE THREAD

**Issue by ConEmu** [ConEmu/old-issues#714]
_From [ckin...@gmail.com](https://code.google.com/u/107158560758466172135/) on September 20, 2012 14:49:30_

The goal is to make it easier to post instructions into blog posts and other HTML pages.  E.g. this looks ugly: http://ceklog.kindel.com/2012/09/18/doing-git-dev-on-two-machines/ But if I could have used HTML like in this jsfiddle for an example: http://jsfiddle.net/ckindel/vjLf8/ it would have been much prettier (and would show off conemu better).

_Original issue: http://code.google.com/p/conemu-maximus5/issues/detail?id=714_

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on September 20, 2012 14:58:38_

Ok, I'll create an option. May be interesting feature.

**Status:** Accepted

**Comment by ConEmu** [comment]
_From [ckin...@gmail.com](https://code.google.com/u/107158560758466172135/) on September 20, 2012 18:15:40_

This will make an interesting test case:  Note line art: http://images.devs-on.net/Image/Q7degKprFqNO4X2x-WindowsPowerShe.png

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on September 28, 2013 12:43:05_

Try build 130928

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on September 28, 2013 12:43:19_

**Status:** Started

**Comment by ConEmu** [comment]
_From [ckin...@gmail.com](https://code.google.com/u/107158560758466172135/) on September 28, 2013 17:53:20_

SWEET! Very nice work.

My only complaint is upon paste the font size is much bigger than in the console. 

Here's a test I did where I pasted into Outlook: http://images.devs-on.net/Image/yplXLD0VG9GDzIfi-Region.png I would expect the font size on the paste to be closer to the original. In my case I am using Source Code Pro at 16px.
